docs/subset15.md

```diff
@@ -418,12 +419,13 @@ Two kinds of `import` are supported.
 
 * `list-constructor-expr` and `mapping-constructor-expr` are not allowed within a `const-expr`.
 * Types in type definitions are restricted semantically, rather than syntactically: a type definition that is referenced from a function definition must define a type that is equivalent to one that can be described using the type-defn grammar in this document. It must also match the type-defn [grammar supported for semantic type-checking](type-subset.md).
+* Once a non-final value is captured it will not be narrowed. If it is already narrowed at the point of capture then capturing will undo the narrowing.
```

Comment:
After a variable is captured, its type can no longer be narrowed (e.g. by a type test) unless the variable is final.

Reply:
Fixed